docs/hub/spaces-sdks-docker-tabby.md

```diff
@@ -0,0 +1,65 @@
+# Tabby on Spaces
+
+[Tabby](https://tabby.tabbyml.com) is an open-source, self-hosted AI coding assistant. With Tabby, every team can set up its own LLM-powered code completion server with ease.
+
+In this guide, you will learn how to deploy your own Tabby instance and use it for development directly from the Huggingface website.
+
+## Your first Tabby Space
+
+In this section, you will learn how to deploy a Tabby Space and use it for yourself or your orgnization.
+
+### Deploy Tabby on Spaces
+
+You can deploy Tabby on Spaces with just a few clicks:
+
+[![Deploy on HF Spaces](https://huggingface.co/datasets/huggingface/badges/raw/main/deploy-to-spaces-lg.svg)](https://huggingface.co/spaces/TabbyML/tabby-template-space?duplicate=true)
+
+You need to define the Owner (your personal account or an organization), a Space name, and the Visibility. To secure the api endpoint, we're configuring the visibility as Private.
+
+![image](https://github.com/huggingface/hub-docs/assets/388154/b00eb7ea-7de6-4832-b3ff-512859bfa92e)
+
+<Tip>
+
+If you want to customize the title, emojis, and colors of your space, go to "Files and Versions" and edit the metadata of your README.md file.
+
+</Tip>
+
+You’ll see the Building status and once it becomes Running your space is ready to go. If you don’t see the Tabby swagger UI, try refreshing the page.
+
+![image](https://github.com/huggingface/hub-docs/assets/388154/f2320e55-7aad-42ed-8b1c-aec4e42e05f0)
```

Comment:
Please add this asset to https://huggingface.co/datasets/huggingface/documentation-images/tree/main/hub and use it from there

Reply:
Merged!